### DISCUSSION TITLE
Confused about background opacity

### DISCUSSION THREAD

**Discussion by IceSentry** [wezterm/wezterm#5962] [Q&A]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

When I set the background opacity to something like 0.5, the main window will not have it's opacity changed. Even if I close it and open it again it doesn't take effect, but if I spawn a new window from the main window the new window will have it's opacity set correctly. I'm not sure what I'm doing wrong but that doesn't seem correct to me.

**Comment by Joe-273** [comment]
I also encountered the same problem
![Snipaste_2024-09-25_02-31-25](https://github.com/user-attachments/assets/90333a21-9cca-4c9b-a1cb-be7c37f4c89c)
